policyengine_uk_data/datasets/frs/frs.py

```diff
@@ -750,9 +750,9 @@ def add_benefit_income(
         0,
     )
 
-    frs["free_school_breakfasts"] = person.FSBVAL
-    frs["free_school_fruit_veg"] = person.FSFVVAL
-    frs["free_school_meals"] = person.FSMVAL
+    frs["free_school_breakfasts"] = person.FSBVAL * 52
```

Comment:
I'm not on review on this, but just saw, and was wondering if we could have a standard of naming numbers like this as a constant before using. 

```suggestion
    WEEKS_IN_YEAR = 52
    frs["free_school_breakfasts"] = person.FSBVAL * WEEKS_IN_YEAR
```